### DISCUSSION TITLE
EnergyPlus desperatley needs a condenser water thermal energy storage object

### DISCUSSION THREAD

**Discussion by lincolnharmer** [NREL/EnergyPlus#9214] [Ideas]
When trying to design an all-electric building HVAC system, condenser water thermal storage is an attractive option. See this recent ASHRAE article for reference: [October2021_12-27_Gill (1).pdf](https://github.com/NREL/EnergyPlus/files/7729534/October2021_12-27_Gill.1.pdf)

Currently E+ doesn't have a thermal storage object which can be added to a condenser water loop as far as I know. Perhaps the chilled water thermal storage object can be used as a work around?

**Comment by bonnema** [comment]
what about trying a user defined plant component?

**Reply by lincolnharmer** [reply]
where is that object documented? I couldn't find it in the input/output reference

**Reply by lincolnharmer** [reply]
sorry, I found it

**Reply by lincolnharmer** [reply]
thanks! this looks a like a possible avenue. This could be impleented in OpenStudio but I will need to add the WaterUse:Storage object via an E+ measure as that isn't supported by OpenStudio directly yet.

**Comment by mjwitte** [comment]
Also, the WaterHeater:* objects can be used for storage. See example files HeatRecovery*.idf. The water heater objects can be a simple storage tank by setting the heater capacity to zero.

**Reply by lincolnharmer** [reply]
thanks, I didn't realize we could use the waterheater objects like this but that makes a lot of sense.
